static/app/components/events/eventTags/eventTagsTree.tsx

```diff
@@ -110,21 +136,105 @@ function TagTreeRow({
       </TreeKeyTrunk>
       <TreeValueTrunk>
         <TreeValue>
-          {originalTag ? (
-            <EventTagsContent
-              tag={originalTag}
+          {originalTag.key === 'release' ? (
+            <VersionHoverCard
               organization={organization}
-              query={generateQueryWithTag(
-                {...location.query, referrer: 'event-tags-tree'},
-                originalTag
-              )}
-              meta={content?.meta ?? {}}
-              {...props}
-            />
+              projectSlug={projectSlug}
+              releaseVersion={content.value}
+              showUnderline
+              underlineColor="linkUnderline"
+            >
+              <Version version={content.value} truncate />
+            </VersionHoverCard>
           ) : (
             content.value
           )}
         </TreeValue>
+        <TreeValueDropdown
+          preventOverflowOptions={{padding: 4}}
+          className={!isVisible ? 'invisible' : ''}
```

Comment:
ultra nit but not negating the ternary
```suggestion
          className={isVisible ? '' : 'invisible'}
```